dot/parachain/statement-distribution/statement_distribution_test.go

```diff
@@ -0,0 +1,28 @@
+package statementdistribution
+
+import (
+	"context"
+
+	parachainutil "github.com/ChainSafe/gossamer/dot/parachain/util"
+)
+
+func CreateStatementDistribution() (*StatementDistribution, chan parachainutil.NetworkBridgeTxMessage) {
+	mockAggregator := parachainutil.NewReputationAggregator(func(rep parachainutil.UnifiedReputationChange) bool {
+		return rep.Type == parachainutil.Malicious
+	})
+
+	subSystemToOverseer := make(chan parachainutil.NetworkBridgeTxMessage, 10)
+
+	return &StatementDistribution{
+		reputationAggregator: mockAggregator,
+		SubSystemToOverseer:  subSystemToOverseer,
+	}, subSystemToOverseer
+}
+
+func CreateChannels() (chan any, chan any, chan any, chan any) {
```

Comment:
```suggestion
func createChannels() (chan any, chan any, chan any, chan any) {
```
Hard to say without seeing this used anywhere, but maybe it makes more sense to create a general test setup function that does most of the repetitive work needed in tests.